### ISSUE TITLE
Does module versioning still allow two artifacts on the same branch to have different backwards compat?

### ISSUE THREAD

**Issue by xorrkaz** [netmod-wg/yang-ver-dt#222]
In yang-semver, we have this text:

```xml
<artwork name="" type="" align="left" alt=""><![CDATA[
      3.5.0 -- 3.6.0 (add leaf foo)
        |
        |
      3.20.0 (added leaf bar)
  ]]></artwork>
<t> In this case, given only the YANG Semver identifiers 3.6.0 and 3.20.0, 
    one would assume that 3.20.0 is backwards compatible with 3.6.0. But in the illegal example above, 3.20.0 is not 
    backwards compatible with 3.6.0 since 3.20.0 does not contain the leaf foo.</t>
<t>Note that this type of branching, where two versions on the same branch have different backwards compatible 
    changes is allowed in <xref target="I-D.ietf-netmod-yang-module-versioning" format="default"/>
    ```

Is this still the case?  That is, would this still be legal -- given our changes -- with module versioning?

**Comment by jsterne** [comment]
I don't see how Module Versioning makes that type of branching illegal (two revisions based on the same parent, each with different BC changes). I think the comment above was specific to YANG Semver where we don't want 3.20.0 to be NBC vs 3.6.0.
Others may feel that branching isn't really *well* supported with just Module Versioning but I think we probably want (need) to allow it (even without YANG Semver).

**Comment by xorrkaz** [comment]
Yeah, I didn't think so, either.  But this text has been in yang-semver for a while.  And I'm going line-by-line, so calling out what I see.  If we think this is allowed, I'll pull out these paragraphs.

**Comment by jsterne** [comment]
I think we should actually keep that in YANG semver though. It isn't allowed for YANG semver.

**Comment by xorrkaz** [comment]
Yes.  I meant just drop the last paragraph.

**Comment by xorrkaz** [comment]
I wasn't sure now that we're taking out the version history bit.  Maybe it is simply because there isn't a concept of branching like with versioning.  It just sounded odd to leave in given where things have landed.
